stagecraft/libs/views/resource.py

```diff
@@ -65,7 +65,9 @@ def list(self, request, **kwargs):
         # Used to filter by, for instance, backdrop user
         filter_args = dict(filter_args.items() + additional_filters.items())
 
-        return self.model.objects.filter(**filter_args).order_by('pk')
+        return self.model.objects.filter(
+            **filter_args).select_related(
+                'data_group', 'data_type').order_by('pk')
```

Comment:
Will this work with models other than dataset? What happens if there's no relationship to data_group and data_type?